tuxemon/states/combat/combat_animations.py

```diff
@@ -71,6 +72,10 @@ def startup(
         graphics: Optional[Mapping[str, str]] = None,
         **kwargs: Any,
     ) -> None:
+        # The defaults are a lie to stop mypy complaining about us violating
```

Comment:
We should by fixed by doing what is proposed in #1167.

Reply:
I could just fix the typehint to make it accurate, even if that makes mypy complain.

It might not be possible though, I've seen exceptions in similar situations.

Reply:
It is fine to let it as is. We will tackle the root of the issue in #1167.